examples/demo/site/scripts/oauth2_consumers_initialize.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+	-- Change the values `TO_COMPLETE' based on your API.
+	-- API SECTET KEY AND API PUBLIC KEY 
+
+INSERT INTO "oauth2_consumers" ("name", "api_secret", "api_key", "scope", "protected_resource_url", "callback_name", "extractor", "authorize_url", "endpoint") 
```

Comment:
Shouldn't we have a admin section to configure those value?
I guess here this is mainly for example.